OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -394,7 +394,7 @@ Action InitInputPanel(Widget panel)
 					{ "CycleBaseKey", "Jump to base" },
 					{ "ToLastEventKey", "Jump to last radar event" },
 					{ "ToSelectionKey", "Jump to selection" },
-					{ "SelectAllUnitsKey", "Select all units on screen" },
+					{ "SelectAllUnitsKey", "Select all military units" },
```

Comment:
Not sure of the context behind this change (i.e. whether we want to change it at all), but the convention is for "combat units" rather than military.